docs/pages/api-reference/liveblocks-react-ui.mdx

```diff
@@ -12,6 +12,201 @@ to build collaborative experiences. Read our
 [Notifications](/docs/get-started/notifications) get started guides to learn
 more.
 
+## AI Copilots
+
+### Default components [#AI-Copilots-Components]
+
+#### AiChat
+
+Displays an interactive AI chat. AI can [use knowledge](#TODO),
+[run actions](#TODO) and [display tools](#TODO).
+
+```tsx
+<AiChat chatId="my-chat-id" />
+```
+
+{/* TODO image */}
+
+Each chat is stored permanently, and is indentified by its unique `chatId`.
+Chats are only visible to the [authenticated user](/docs/authentication) who
+created the chat.
+
+```tsx
+import { AiChat } from "@liveblocks/react-ui";
+
+function Chat() {
+  return <AiChat chatId="my-chat-id" />;
+}
+```
+
+##### Show placeholder content in new chats
+
+In chats without messages, you can display placeholder content to guide the
+user...
+
+{/* TODO the API we figure out */}
+
+{/* TODO show initial suggestions if it's supported */}
+
+##### Adjusting the chat’s width
+
+{/* TODO add in name of layout below */}
+
+When using the default `TODO` layout, it’s possible to adjust the chat’s width
+by setting the `--lb-ai-chat-container-width` CSS variable. This allows the
+chat’s scroll window to stay full width, whilst keeping the composer and
+messages centered in the middle.
+
+```css
+.lb-root {
+  --lb-ai-chat-container-width: 600px;
+}
+```
+
+You can also set this property directly on the `AiChat` component.
```

Comment:
We could also have a Tailwind CSS snippet, `className="[--lb-ai-chat-container-width:600px]"`